mqtt-streaming/src/test/scala/docs/scaladsl/MqttSessionSpec.scala

```diff
@@ -763,12 +794,19 @@ class MqttSessionSpec
       session ! Command(publish)
 
       server.expectMsg(publishBytes)
+      // This reply triggers an error showing in the logs which hinders proper shutdown
+      //   7   │ 2019-03-06 11:28:23,035 ERROR [mqtt-spec-akka.actor.default-dispatcher-3] [akka.actor.OneForOneStrategy]  56 (of class java.lang.Integer)
+      //   8   │ scala.MatchError: 56 (of class java.lang.Integer)
+      //   9   │     at akka.stream.impl.fusing.GraphInterpreter.$anonfun$toSnapshot$4(GraphInterpreter.scala:662)
```

Comment:
This indicates some error in an MQTT graph stage - I haven't gone deeper, yet.
cc @huntc

Reply:
Thanks! I shall look at this first thing.